docs/content/concepts/ops-jobs-graphs/op-events.mdx

```diff
@@ -113,6 +113,8 @@ to keep track of can be considered an asset.
 
 Generally, you'd want to send this event directly after you persist the asset to your external system. All <PyObject object="AssetMaterialization" /> events must define an `asset_key`, which is a unique identifier to describe the asset you are persisting.
 
+If you're using [software-defined assets](/concepts/assets/software-defined-assets), you don't need to explicitly record these events – the framework handles it for you.
```

Comment:
```suggestion
If you're using [Software-defined Assets](/concepts/assets/software-defined-assets), you don't need to record these events explicitly – the framework handles it for you.
```

Updating SDA as per our style guide; suggesting a fix for un-splitting an infinitive